### PR TITLE
Change maintainer for resubmission as 0.3.9

### PR DESCRIPTION
**Pull request by kuriwaki**
Mostly clearing out some things that we don't seem to be using. 

As for the concern with readr 2.0.0 (https://github.com/OuhscBbmc/REDCapR/issues/343), I ran the local tests after updating 2.0.0 and I don't see any issues. After rethinking I don't think we need to set >= 2.0.0 in `DESCRIPTION` -- the modal dataverse file is a TSV, and our default `readr::read_tsv` should work regardless of v2 or v1. @wibeasley  does that sound right?

### CODE REVIEW

File: for-developers/developer-tasks.R [hunk 1]
```diff
@@ -22,18 +17,14 @@ gp <-
 goodpractice::results(gp)
 gp
 
-devtools::run_examples(); #dev.off() #This overwrites the NAMESPACE file too
-# devtools::run_examples(, "redcap_read.Rd")
+devtools::run_examples(); # dev.off() # This overwrites the NAMESPACE file too
 test_results_checked <- devtools::test()
-# test_results_checked <- devtools::test(filter = "read_b.*")
```
Comment (wibeasley) [inline for-developers/developer-tasks.R:28]: @kuriwaki, consider keeping this line.  There are a bunch of times where I want to isolate and test a single file.  This line gives provides a headstart.

File: Makefile [hunk 2]
```diff
@@ -1,39 +0,0 @@
-pkg = $(shell basename $(CURDIR))
-
-all: build
-
-NAMESPACE: R/*
-	Rscript -e "devtools::document()"
-
-README.md: README.Rmd
```
Comment (wibeasley) [inline Makefile:8]: @kuriwaki, it's fine if you delete the Makefile.  None of my other packages use them.  But you'll need to remember to manually knitr README.Rmd to update README.md.

That tradeoff is probably worth it to you.  I just wanted to state the new manual requirement.

File: for-developers/notes-datafest-20201.md [hunk 3]
```diff
@@ -1,37 +0,0 @@
-Notes for Datafest 2021 Workshop,
```
Comment (wibeasley) [inline for-developers/notes-datafest-20201.md:1]: @kuriwaki, I wouldn't delete this.  It's tucked away in GitHub and not interfering related to R.  (Notice it's listed in .Rbuildignore: https://github.com/IQSS/dataverse-client-r/blob/d19b2410ed31867df5d8341bd7471bce6823ce15/.Rbuildignore#L3)

Or if you do delete it, replace it with something similar for new people to get a quick feel for the package.

**Comment by wibeasley** [comment]
@kuriwaki, agreed.  If it doesn't seem to matter, don't add it as a requirement.  But I'd consider adding the requirement after a reasonable time for everyone to update.  Say, a year?

This is because we'll always need keep this dichotomy in mind until there's a single parser engine.  If there's a weird bug that sometimes works and sometimes fails, remember that one user's machine may be running the old readr engine, while another's is running the new readr/vroom engine.

**Comment by kuriwaki** [comment]
Sounds good about clarification in the  long term. 

Though actually, our win-devel gh actions check is consistently failing and the error logs do suggest it might be readr 2.0.0 related.

**Comment by wibeasley** [comment]
Does the win-builder error look like the GitHub Actions error?  
https://github.com/IQSS/dataverse-client-r/runs/3129414663?check_suite_focus=true#step:8:3202
> x Failed to build source package 'readr' 

If so, I'm not sure you can do much, at least tonight.  It looks like it's correctly retrieving readr 2.0, but I guess not building it.  This is a few steps before anything to do with the dataverse package.

https://github.com/IQSS/dataverse-client-r/runs/3129414663?check_suite_focus=true#step:8:62

I bet it's some CRAN binary problem that will be worked out by tomorrow.

I had a smooth submission with REDCapR tonight, which also has a the readr 2.0 prereq.  So that's a good sign for this.

If you want me to look at the win-builder error, you can post the url.  It stays on their server for a day or two.

**Comment by kuriwaki** [comment]
Ok good to hear about your RedcapR success.  I was referring to only the githb Actions error, but I did run `devtools::check_win_devel()` after your note and got an error with the vignette:


```
Error in (function (command = NULL, args = character(), error_on_status = TRUE,  : 
  System command 'R' failed, exit status: 1, stdout + stderr (last 10 lines):
E> Quitting from lines 44-45 (C-download.Rmd) 
E> Error: processing vignette 'C-download.Rmd' failed with diagnostics:
E> Not Found (HTTP 404). Failed to API endpoint does not exist on this server. Please check your code for typos, or consult our API guide at http://guides.dataverse.org..
E> --- failed re-building ‘C-download.Rmd’
E> 
E> SUMMARY: processing the following file failed:
E>   ‘C-download.Rmd’
E> 
E> Error: Vignette re-building failed.
E> Execution halted
```

**Comment by wibeasley** [comment]
@kuriwaki, I'm guessing that it was a temporarily outage with the win-builder server?  It's running on my two local machines, and on on GitHub actions (r release: https://github.com/IQSS/dataverse-client-r/pull/104/checks?check_run_id=3129414679).

The failing build on Github Actions (R development) two days looks like it's related to the version of readr released the day before.  I bet it's fine if you run it now.  https://github.com/IQSS/dataverse-client-r/pull/104/checks?check_run_id=3129414663#step:8:3202

**Review by kuriwaki — COMMENTED** [review]
@wibeasley thank you for the comments. I brought back the talk notes with some updates, brought back `devools::test` with filter, and kept the makefile removed. I noticed the makefile referenced outdated vignettes.  

(I started my own "review" by accident in trying to reply - and could not undo)

**Comment by kuriwaki** [comment]
I will try to resubmit this as soon as rhub and Actions win_devel tests check off. 

The `readr` installation failure is now gone but seems to be replaced by a `tibble` installation failure. (tibble is only [used](https://github.com/IQSS/dataverse-client-r/blob/8a55de22170608b502693b7d63ac4455e719e51e/vignettes/C-download.Rmd#L68-L73) in a vignette when displaying a dowlonaded file).

I created a checklist #105 following usethis 2.0.1 (`use_release_issue`) but I may not complete all the new items it for now given that the priority should be to get it back on CRAN as it was previously.

**Pull request merged**